src/gdf_table.cuh

```diff
@@ -1078,29 +1273,51 @@ private:
             typename size_type>
   gdf_error gather_column(column_type * const __restrict__ input_column,
```

Comment:
This function looks to be redundant now with your new `gather_column` function that accepts a `gdf_column` as the input and output columns